I'm not sure why the test shows this error 
`src/TrackingTools/TrackFitters/interface/DebugHelpers.h:19:7: error: 'LogTrace' was not declared in this scope`
I don't get this error while compiling locally

--- a/TrackingTools/TrackFitters/interface/DebugHelpers.h
+++ b/TrackingTools/TrackFitters/interface/DebugHelpers.h
@@ -12,6 +12,7 @@
 #include "DataFormats/MuonDetId/interface/ME0DetId.h"
 #include "DataFormats/MuonDetId/interface/MuonSubdetId.h"
 #include "TrackingTools/GsfTools/interface/GetComponents.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
 
 namespace {
   inline void dump(TrackingRecHit const& hit, int hitcounter, const std::string& msgCat) {

